src/MCPClient/lib/clientScripts/policyCheck.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+import json
+import os
+import shutil
+import sys
+
+from custom_handlers import get_script_logger
+
+import django
+django.setup()
+from fpr.models import FPRule, FormatVersion
+from main.models import File, SIP
+
+from executeOrRunSubProcess import executeOrRun
+import databaseFunctions
+from dicts import replace_string_values
+
+# Note that linkTaskManagerFiles.py will take the highest exit code it has seen
+# from all tasks and will use that as the exit code of the job as a whole.
+SUCCESS_CODE = 0
+NOT_APPLICABLE_CODE = 0
+FAIL_CODE = 1
+
+
+class PolicyChecker:
```

Comment:
Yes, you're right. I'm used to writing Python3-only code where subclassing `object` is redundant.